packages/@cdklabs/cdk-cicd-wrapper/test/stacks/PostDeployExecutorStack.test.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as cdk from 'aws-cdk-lib';
+import { Template } from 'aws-cdk-lib/assertions';
+import { ResourceContext, Stage } from '../../src/common';
+import { PostDeployExecutorStack } from '../../src/stacks';
+import { TestAppConfig } from '../TestConfig';
+
+describe('post-deploy-role-stack-test', () => {
+  const app = new cdk.App();
+
+  new ResourceContext(app, new cdk.Stack(), { ...TestAppConfig, resourceProviders: {} });
+  ResourceContext.instance().initStage(Stage.DEV);
+
+  const template = Template.fromStack(
+    new PostDeployExecutorStack(app, 'PostDeployExecutorStack', {
+      env: TestAppConfig.deploymentDefinition.DEV.env,
+      resAccount: TestAppConfig.deploymentDefinition.RES.env.account,
+      stageName: Stage.DEV,
+      name: TestAppConfig.applicationName,
+    }),
+  );
+
+  test('Check if role exists', () => {
+    template.resourceCountIs('AWS::IAM::Role', 1);
+    template.hasResourceProperties('AWS::IAM::Role', {
+      AssumeRolePolicyDocument: { Statement: [{ Action: 'sts:AssumeRole', Effect: 'Allow' }] },
+    });
+  });
+
+  test('Check if output exists', () => {
+    template.hasOutput('RoleArnCfnOutput', {
+      Value: 'arn:aws:iam:eu-west-1:234567890123:role/post-deploy-234567890123-eu-west-1-VanillaPipeline-DEV',
```

Comment:
Please remove the VanillaPipeline value, you can use something else more neutral